docs/src/main/paradox/scala/http/server-side/low-level-api.md

```diff
@@ -47,7 +47,7 @@ Arguments to the `Http().bind` method specify the interface and port to bind to
 incoming HTTP connections. Additionally, the method also allows for the definition of socket options as well as a larger
 number of settings for configuring the server according to your needs.
 
-The result of the `bind` method is a @scala[`Source[Http.IncomingConnection]]]@java[@unidoc[Source[Http.IncomingConnection]]] which must be drained by the application in
+The result of the `bind` method is a @scala[@unidoc[Source[Http.IncomingConnection]]]@java[@unidoc[Source[Http.IncomingConnection]]] which must be drained by the application in
```

Comment:
Could be merged if the unidoc link works at all for inner classes?

Reply:
Currently it just links to `Source` in this case, but indeed it would be cool and entirely possible to support linking to the individual type parameters as well

Reply:
Let's add tickets for those improvements.